<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QFieldCloudDialog</class>
 <widget class="QDialog" name="QFieldCloudDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>574</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QFieldCloud</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="loginFormGroup">
     <property name="title">
      <string>QFieldCloud Login</string>
     </property>
     <layout class="QFormLayout" name="formLayout_3">
      <item row="0" column="1">
       <widget class="QLabel" name="loginFeedbackLabel">
        <property name="visible">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="usernameLabel">
        <property name="text">
         <string>Username</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="usernameLineEdit">
        <property name="text">
         <string>demo_user</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="passwordLabel">
        <property name="text">
         <string>Password</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QgsPasswordLineEdit" name="passwordLineEdit">
        <property name="text">
         <string>demo_pwd</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <layout class="QHBoxLayout" name="loginFormButtonsLayout" stretch="0,0,0,0">
        <item>
         <widget class="QCheckBox" name="rememberMeCheckBox">
          <property name="text">
           <string>Remember Me</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="loginFormButtonsSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="loginButton">
          <property name="text">
           <string>Login</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="logoutButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Logout</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="1">
       <widget class="QLabel" name="syncFeedbackLabel">
        <property name="visible">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="projectsStackGroup">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="title">
      <string>Projects</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QStackedWidget" name="projectsStack">
        <property name="inputMethodHints">
         <set>Qt::ImhDialableCharactersOnly</set>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="projectsListPage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="2" column="1">
           <layout class="QHBoxLayout" name="projectsSelectButtonLayout" stretch="0,0,0,0,0">
            <item>
             <spacer name="projectsSelectButtonSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="createButton">
              <property name="text">
               <string>Create</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="editButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Edit</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="deleteButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Delete</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="refreshButton">
              <property name="text">
               <string>Refresh</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="projectsFeedbackLabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QTableWidget" name="projectsTable">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>60</number>
            </attribute>
            <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
             <bool>true</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <column>
             <property name="text">
              <string>Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Owner</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Private</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Status</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Synchronize</string>
             </property>
            </column>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="projectsFormPage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="2" column="0">
           <widget class="QLabel" name="projectNameLabel">
            <property name="text">
             <string>Name</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="projectNameLineEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="projectDescriptionLabel">
            <property name="text">
             <string>Description</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPlainTextEdit" name="projectDescriptionTextEdit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="projectOwnerLabel">
            <property name="text">
             <string>Owner</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <layout class="QHBoxLayout" name="projectFormButtonLayout">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QCheckBox" name="projectIsPrivateCheckBox">
              <property name="text">
               <string>Private</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="projectFormButtonSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="backButton">
              <property name="text">
               <string>Back</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="submitButton">
              <property name="text">
               <string>Submit</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="projectUrlLabelValue">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="projectUrlLabel">
            <property name="text">
             <string>Project URL</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QComboBox" name="projectOwnerComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsPasswordLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qgspasswordlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
